build/configs/imxrt1020-evk/scripts/flash-dtcm.ld

```diff
@@ -57,7 +57,7 @@ MEMORY
 	flash (rx)  : ORIGIN = 0x60000000, LENGTH = 0x00800000
 	sram  (rwx) : ORIGIN = 0x20200000, LENGTH = 0x00020000
 	itcm  (rwx) : ORIGIN = 0x00000000, LENGTH = 0x00010000
-	dtcm  (rwx) : ORIGIN = 0x20000000, LENGTH = 0x00010000
+	dtcm  (rwx) : ORIGIN = 0x20000000, LENGTH = 0x00020000
```

Comment:
Total size of internal ram is 256KB. If you want to increase dtcm size, you have to decrease sram size.